packages/react-composites/src/composites/localization/locales/ru-RU/strings.json

```diff
@@ -315,8 +315,10 @@
       "startSpotlightCancelButtonLabel": "Отмена",
       "stopSpotlightHeading": "Больше не рекомендовать это видео?",
       "stopSpotlightOnSelfHeading": "Выйти из работы с важным?",
+      "stopAllSpotlightHeading": "Отменить все рекомендации видео?",

```

Comment:
It doesn't have the same meaning as in English...

Reply:
Created an insident for this

Reply:
> Created an incident for this

Thanks for checking.